src/solc_0.7/common/BaseWithStorage/WithMinter.sol

```diff
@@ -5,21 +5,20 @@ import "./WithAdmin.sol";
 
 contract WithMinter is WithAdmin {
     address internal _minter;
-    event MinterChanged(address oldMinter, address newMinter);
 
-    modifier onlyMinter() {
-        require(msg.sender == _minter, "MINTER_ACCESS_DENIED");
-        _;
-    }
+    /// @dev Emits when the Minter address is changed
```

Comment:
we should not remove `onlyMiner` modifier

Reply:
nope. restoring now. Not actually sure why I removed this.